examples/paul_graham_essay/InsertDemo.ipynb

```diff
@@ -195,7 +195,7 @@
     "# NOTE: we truncate to the first 30 nodes to save on cost\n",
     "document = SimpleDirectoryReader('data').load_data()[0]\n",
     "text_splitter = TokenTextSplitter(separator=\" \", chunk_size=256, chunk_overlap=20)\n",
-    "text_chunks = text_splitter.split_text(document.text)\n",
+    "text_chunks = text_splitter.split_text(document.get_text())\n",
```

Comment:
my fork may be behind, but is this the way to get text from docs from now on?

Reply:
Yep! I thought about it for a bit and thought having an explicit getter (which would fail if the text field did not exist), was the best compromise in order to work with dataclasses. That way we still have the power of the dataclass (you can create objects without defining __init__ methods, serialize them, etc.), but we can still have getters that will return the value itself and not the optional value